keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/plugins/ResourceHandler.kt

```diff
@@ -64,7 +68,7 @@ abstract class ResourceHandler<S : ResourceSpec, R : Any>(
    * @param resource a fully-resolved version of the persisted resource spec. You can assume that
    * [applyResolvers] has already been called on this object.
    */
-  protected abstract suspend fun toResolvedType(resource: Resource<S>): R
+  protected abstract suspend fun toConcreteType(resource: Resource<S>): C
```

Comment:
I renamed this to remove any ambiguity with the `resolve` function implemented in handlers and the ad hoc differ. We now make a distinction between a "resolved resource" being just a resource that has passed by resolvers, and the "concrete type" that is the output of a handler translating a resource/spec into a diff-able type.